cfg.py.examples

```diff
@@ -50,3 +50,19 @@ sPlayer = "/usr/bin/mpv"
 sPlayerArgs = "--title='aiostreams' --really-quiet"
 aPlayer = "/usr/bin/mpv"
 aPlayerArgs = "--title='aiostreams' --really-quiet"
+
+# Windows10 For "C:\Program Files (x86)" VLC player
```

Comment:
Is it necessary? Who actually uses Windows other than Gaming?

Reply:
People use it for coding as well xD
 
Also a bit of radio instead of having to put music by ourselves is better :)

Reply:
Thank you so much for your contribution. This will be included in the next release.